content/posts/blog/Return_of_experience_on_XMPP_meetup_in_Krakow.md

```diff
@@ -0,0 +1,31 @@
+In May of 2017 a meetup was organized in Krakow, Poland. We announced it on the [XSF blog](https://xmpp.org/2017/04/first-krakow-xmpp-meetup/), and [on meetup.com](htpps://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/) (also a few social networks).  The reponse from the XMPP/Jabber community was very positive, as we were fresh off the heels of FOSDEM and many members had just recently met at the XMPP Summit.
+
+## Philosophy & goals
+
+Firstly, it was a cooperation between Tigase and MongooseIM. Both organizations provide open source XMPP server software and bring professional expertise on the protocol.  However, we wanted it to be a humble and fast organisation, rather than a meticulously planned event. Calling to mind the addage "done is better than perfect".  The ultimate goal of the meetup was to meet colleagues of the XMPP and Jabber community, brainstorm, and overall improve communication between users and developers of the protocol.  An additional benefit from the meeting so soon after the Summit was to strengthen the trend and popularity of XMPP meetups in the European Union.
```

Comment:
> bring professional expertise on the protocol

Bring professional expertise to what?

> a humble and fast organisation

What does it mean for an organization to be humble or fast?

"organization"

> Calling to mind the addage "done is better than perfect".

This is not a complete sentence.